app/client/components/identity/models.ts

```diff
@@ -38,7 +38,7 @@ export interface User {
   postcode: string;
   country: string;
   countryCode: string;
-  localNumber: string;
+  localNumber: number;
```

Comment:
would this affect the input validation or anything like that ?

Reply:
It shouldn't, the TypeScript types should only be used at compile time

Reply:
sorry for some reason I thought you were changing this the other way areound,  why does this need to be a number here ?

Reply:
this represents the type of the form input field, which is numeric
Before, it was set to string which is incorrect